developer-docs-site/docs/transactions/basics-life-of-txn.md

```diff
@@ -53,68 +53,68 @@ For the relevant steps, we've included a link to the corresponding inter-compone
 
 ## Committing the block
 
 | Description                                                  | Aptos Node Component Interactions                             |
 | ------------------------------------------------------------ | ------------------------------------------------------------ |
-| 12. **Consensus → Execution**, **Execution → Storage **:If the proposed block's execution result is agreed upon and signed by a set of validators that have the quorum of votes, validator V<sub>1</sub>'s execution component reads the full result of the proposed block execution from the speculative execution cache and commits all the transactions in the proposed block to persistent storage with their results. | [4. Consensus](#4-consensus-→-execution), [3. Execution](#3-consensus-→-execution), [4. Execution](#4-execution-→-storage), [3. Storage](#3-execution-→-storage) |
+| 12. **Consensus → Execution**, **Execution → Storage**: If the proposed block's execution result is agreed upon and signed by a set of validators that have the quorum of votes, validator V<sub>1</sub>'s execution component reads the full result of the proposed block execution from the speculative execution cache and commits all the transactions in the proposed block to persistent storage with their results. | [4. Consensus](#4-consensus--execution), [3. Execution](#3-consensus--execution), [4. Execution](#4-execution--storage), [3. Storage](#3-execution--storage) |
 
 Alice's account will now have 100 Aptos Coins, and its sequence number will be 6. If T<sub>5</sub> is replayed by Bob, it will be rejected as the sequence number of Alice's account (6) is greater than the sequence number of the replayed transaction (5).
 
 # Aptos node component interactions
 
-In the [previous section](#lifecycle-of-the-transaction), we described the typical lifecycle of a sample transaction from being submitted to being committed to the Aptos Blockchain's distributed database. Now let's look in more depth at the inter-component interactions of Aptos nodes as the blockchain processes transactions and responds to queries. This information will be most useful to those who:
+In the [previous section](#lifecycle-of-the-transaction), we described the typical lifecycle of a transaction (from transaction submission to transaction commit). Now let's look at the inter-component interactions of Aptos nodes as the blockchain processes transactions and responds to queries. This information will be most useful to those who:
 
-* Would like to get an overall idea of how the system works under the covers.
-* Are interested in eventually contributing to the Aptos-core software.
+* Would like to get an idea of how the system works under the covers.
+* Are interested in eventually contributing to the Aptos Blockchain.
```

Comment:
this was actually intentional, but I guess we're still a ways away from distinguishing the two.